packages/webcert/src/store/certificate/certificateMiddleware.ts

```diff
@@ -317,25 +322,94 @@ const handleStartSignCertificate: Middleware<Dispatch> = ({ dispatch, getState }
 
   const signingMethod = getState().ui.uiUser.user.signingMethod
 
-  if (signingMethod === SigningMethod.FAKE) {
-    dispatch(fakeSignCertificate)
-  } else if (signingMethod === SigningMethod.DSS) {
+  switch (signingMethod) {
```

Comment:
Do we have any unittests for this? :)

Reply:
Don't think there are any currently, I'll look into adding that.

Reply:
unit tests have been added.